RELEASE_NOTES.md

```diff
@@ -1,6 +1,18 @@
 # Git Town Release Notes
 
-## Unreleased
+## 7.3.0 (2019-05-06)
+
+#### New Features
+
+- pruning perennial branches
+- support branch names with special characters
```

Comment:
I'd say both of these are bug fixes. The first should also become something more like: `prune branches now properly update perennial branch config`

Reply:
okay